### ISSUE TITLE
need build new v1.4.0 image with golang 1.11.4

### ISSUE THREAD

**Issue by mytxyang** [kubernetes/kube-state-metrics#628]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> /kind feature

**What happened**:
we need kube-state-metrics v1.4.0 to upgrade with golang 1.11.4 to fix 3 security vulnerabilities:
https://groups.google.com/forum/#!topic/golang-announce/Kw31K8G7Fi0

Currently `gcr.io/google_containers/kube-state-metrics:v1.4.0` cannot pass security vulnerabilities.

**What you expected to happen**:
Please help to build `gcr.io/google_containers/kube-state-metrics:v1.4.0` with golang 1.11.4.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):1.11.0
- Kube-state-metrics image version: v1.4.0

**Comment by brancz** [comment]
I don't think this is necessary, the only place kube-state-metrics could be affected could be the x509 vulnerability, in which case this would mean that you are provisioning certificates exploiting this vulnerability. We will build v1.5.0 with v1.11.4, which will be released soon.

**Comment by mytxyang** [comment]
Thanks, could you please show the detail time that can deliver v1.5.0?

**Comment by brancz** [comment]
the pull request to cut it is already out :slightly_smiling_face: https://github.com/kubernetes/kube-state-metrics/pull/629
